docs/source/model/signatures.rst

```diff
@@ -748,7 +748,51 @@ model signatures in log_model calls when signatures aren't specified.
     Secondly, and just as importantly, it **validates the model's requirements**. This input example is utilized to execute a prediction 
     using the model that is about to be logged, thereby enhancing the accuracy in identifying model requirement dependencies.
     It is **highly recommended** to always include an input example along with your models when you log them.
+    Since MLflow 2.16.0, if the provided input example fails on validation, a warning is issued with an instruction to validate the input example
```

Comment:
nit: not sure if we have to give a notice of "we will emit warning" here, as the warning itself is pretty self-explanatory...?